spec/bson/int64_spec.rb

```diff
@@ -71,6 +71,25 @@
     it_behaves_like "a bson element"
     it_behaves_like "a deserializable bson element"
 
+    context 'canonical deserialization' do
```

Comment:
A similar test should be added for BSON::Symbol::Raw, I think it would probably have to go into symbol_spec because `from_bson`  would be called on Symbol not BSON::Symbol::Raw.